collection-scripts/gather_network_logs

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+BASE_COLLECTION_PATH="must-gather"
+NETWORK_LOG_PATH="${BASE_COLLECTION_PATH}/network_logs/"
+
+mkdir -p ${NETWORK_LOG_PATH}/
+
+NODES=$(oc get nodes --no-headers -o custom-columns=":metadata.name")
```

Comment:
Pleas create an input to the script that takes a string of space delimited node names. 

If nothing is supplied (this command can be run); and a warning should be printed to the user that this is not recommended (to run the script without node names) and that it may take a large amount of time (print how many nodes were collection information from)!

Reply:
makes sense